analysis_driver/client.py

```diff
@@ -41,6 +46,10 @@ def main():
             scanner.get_dataset(d).reset()
         for d in args.force:
             scanner.get_dataset(d).force()
+        for d in args.stop:
+            pid = scanner.get_dataset(d).most_recent_proc.get('pid')
+            if pid:
+                os.kill(pid, 15)
```

Comment:
Can we change the signal to 10 (USR1) to have something specific to analysis driver so we can distinguish between process killed by analysis driver or by something else